CoastalResilienceNetwork/GeositeFramework
# Consolidate stylesheets, remove unused JS and CSS libraries

This PR consolidates the two main stylesheets used in the framework (app.css, from the old design, and main.css, from the new design). It also fixes a few other small UI issues.

**Testing**
- Launch the app. It may be easier to close Visual Studio, reopen it, and then open the solution file. Several files are removed, which sometimes causes Visual Studio to get tripped up when it tries to update.
- Browse around the app, and verify that components of the app don't look unstyled.
- Verify the the app title and subtitle are not links.
- Use the measure tool, and verify that the tooltips are readable and that the completed tooltip doesn't have a blue background image.

![image](https://cloud.githubusercontent.com/assets/1042475/19737378/34d53d2c-9b81-11e6-878c-9734f75f1cfc.png)


**Notes**
- The Dijit/Dojo controls used by most plugins work intermittently. I haven't really spent any time tracking down why this is, but sometimes they work, and sometimes they don't.
- The export modal style is messed up because it relied on some of the base Foundation classes. I've add this to #690.

Connects to #691
